src/views/portfolio/vulnerabilities/VulnerabilityCreateVulnerabilityModal.vue

```diff
@@ -441,6 +625,106 @@ export default {
           {text: this.$i18n.t('message.cvss_low'), value: 'L'},
           {text: this.$i18n.t('message.cvss_high'), value: 'H'}
         ]
+      },
+      owaspOptions: {
+        sl: [
+          {text: this.$i18n.t('message.owasp_rr_skill_level_none'), value: '1'},
+          {text: this.$i18n.t('message.owasp_rr_skill_level_some'), value: '3'},
+          {text: this.$i18n.t('message.owasp_rr_skill_level_advanced'), value: '5'},
+          {text: this.$i18n.t('message.owasp_rr_skill_level_network_and_programming'), value: '6'},
+          {text: this.$i18n.t('message.owasp_rr_skill_level_security_penetration_testing'), value: '9'}
+        ],
+        m: [
+          {text: this.$i18n.t('message.owasp_rr_motivation_low'), value: '1'},
+          {text: this.$i18n.t('message.owasp_rr_motivation_possible_reward'), value: '4'},
+          {text: this.$i18n.t('message.owasp_rr_motivation_high_reward'), value: '9'}
+        ],
+        o: [
+          {text: this.$i18n.t('message.owasp_rr_opportunity_full'), value: '0'},
+          {text: this.$i18n.t('message.owasp_rr_opportunity_special'), value: '4'},
+          {text: this.$i18n.t('message.owasp_rr_opportunity_some'), value: '7'},
+          {text: this.$i18n.t('message.owasp_rr_opportunity_none'), value: '9'},
+        ],
+        s: [
+          {text: this.$i18n.t('message.owasp_rr_threat_size_dev_sa'), value: '2'},
+          {text: this.$i18n.t('message.owasp_rr_threat_size_intranet'), value: '4'},
+          {text: this.$i18n.t('message.owasp_rr_threat_size_partners'), value: '5'},
+          {text: this.$i18n.t('message.owasp_rr_threat_size_auth_users'), value: '6'},
+          {text: this.$i18n.t('message.owasp_rr_threat_size_anonymous_internet_users'), value: '9'}
+        ],
+        lc: [
+          {text: this.$i18n.t('message.owasp_rr_loss_of_confidentiality_minimal_non_sensitive'), value: '2'},
+          {text: this.$i18n.t('message.owasp_rr_loss_of_confidentiality_minimal_critical'), value: '6'},
+          {text: this.$i18n.t('message.owasp_rr_loss_of_confidentiality_extensive_non_sensitive'), value: '6'},
```

Comment:
It's a bit odd to have multiple options map to the same factors. I thought it was a UI glitch at first 😅

![image](https://user-images.githubusercontent.com/5693141/207653810-4bf6e16a-0349-4a8c-be68-7efaa25be4db.png)

Would it be possible to move those into a single option field?

Reply:
Done